src/components/Header/index.tsx

```diff
@@ -27,7 +40,16 @@ export function Header() {
           </Heading>
         </Flex>
         <Box width={0.3}>
-          <Input id="search" name="search" placeholder="search" />
+          <form onSubmit={onSearchSubmit}>
+            <Input
+              id="search"
+              name="search"
+              onChange={onSearchChange}
+              placeholder="search"
+              value={searchValue}
+            />
+            {searchValue}
```

Comment:
just a nitpick but it seems a little odd mirroring the searchValue right under the search bar - could we remove this?

Reply:
Oh whoops yes this was for debugging purposes. Out it comes!